Jenkinsfile

```diff
@@ -285,7 +292,13 @@ def generateStep(Map params = [:]){
         dir("${BASE_DIR}"){
           retry(2){
             sleep randomNumber(min:10, max: 30)
-            sh(label: "Run Tests", script: """.ci/scripts/test.sh "${version}" "${tav}" "${edge}" """)
+            if (version?.startsWith('6')) {
+              catchError {
```

Comment:
@v1v Why differ the execution for Node.js 6 here?

Reply:
Fair point, we have discussed this in the following comment:
- https://github.com/elastic/apm-agent-nodejs/pull/1231#issuecomment-514514260

The idea was as node.js 6 seems to be flaky and decided that rather than failing the whole pipeline to set the build as unstable if anything happens with nodejs6.

Jenkins pipelines provide three major different status:
- Stable
- Unstable
- Failure

With this implementation, the build status for nodejs6 in case it fails, would be unstable rather than failure.

Does it make sense?